components/NewPersonView/Layout.tsx

```diff
@@ -84,16 +84,18 @@ const Layout = ({ person, children }: Props): React.ReactElement => {
     });
 
   const secondaryNavigation = [
-    {
-      text: 'Add case note',
-      href: `/people/${person.id}/case-note`,
-    },
     {
       text: 'Add warning note',
       href: `/people/${person.id}/warning-notes/add?id=${person.id}`,
     },
   ];
 
+  if (person.contextFlag === 'C')
```

Comment:
I really dislike that we determine in the component if a link to a form is displayed or not compared to our boolean flags.

As we saw before multiple forms got deployed when they shouldn't have been and I had to rewrite that section of code to tidy it up and make it manageable.

Reply:
There is also no test to determine if this can be seen or not depending on groups the user belongs to!